src/Dropdown/dropdown.less

```diff
@@ -148,6 +148,11 @@
 
       &:hover {
         background: #f5f5f6;
+
+        .inverse {
+          background: #51596a;
+        }
```

Comment:
Let's change how this is defined. We don't want to have to set the `.inverse` class on all elements within the dropdown, just the `.dropdown` element itself.

So let's write something like:
```CSS
.dropdown {
  &.inverse {
    li {
      &.is-selectable {
        ...
      }
    }
  }
}
```